products/sle/main.pm

```diff
@@ -497,11 +501,13 @@ sub load_inst_tests() {
             loadtest "installation/disk_space_fill";
         }
     }
-    if (get_var('SCC_REGISTER', '') eq 'installation') {
-        loadtest "installation/scc_registration";
-    }
-    else {
-        loadtest "installation/skip_registration";
+    if (!check_var('VERSION', '15')) { # No registration in SLE 15 atm - rbrown 04/07/17
+        if (get_var('SCC_REGISTER', '') eq 'installation') {
```

Comment:
I guess @rwx788 just proposed to use the logical equivalent `check_var('SCC_REGISTER', 'installation')`